modules/sdp/SDP.js

```diff
@@ -20,8 +21,9 @@ export default class SDP {
      *
      * @param {string} sdp - The SDP generated by the browser when SDP->Jingle conversion is needed, an empty string
      * when Jingle->SDP conversion is needed.
+     * @param {boolean} isP2P - Whether this SDP belongs to a p2p peerconnection.
      */
-    constructor(sdp) {
+    constructor(sdp, isP2P = false) {
```

Comment:
The `isP2P` option feels weird to me. From the prespective of a class that works with SDP, it's probably best to name it by the behavior it encapsulates, rather than when you want to actually use that behavior.

If my reading is correct something along the lines of `useMid` or bundleGroupIsMid` would do?

Reply:
Now that we are handling the many m-lines to one media description conversion in SDP class itself, we need to know if the SDP is for p2p or not. It is not just for bundle group line, in the follow up PR, that option is used to do Jingle->SDP differently for P2P vs JVB SDPs.

Reply:
Ack.